web/src/app/shared/configuration/tests/configuration.ts

```diff
@@ -0,0 +1,120 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import { getControlType, getKey, getOptions, getValidator, getValue, TFormOptions, IOutput } from '../field.service';
+
+import { IConfig, IConfigAttr, IFieldStack, ILimits, IUIoptions, stateType, TNForm, TValue } from '../types';
+import { IYContainer, IYField, IYspec } from '../yspec/yspec.service';
+
+export class YContainer {}
+
+export class YField {}
```

Comment:
It seems useless

Reply:
it' todo list

Reply:
Please, comment it like 'TODO' in the code